docs/security/index.rst

```diff
@@ -25,5 +25,6 @@ Kyuubi Security Overview
 
     Authentication <authentication>
     kinit
+    proxyuser
```

Comment:
proxyuser seems not to be relevant?

Reply:
Renamed it to hadoop_credentials_manager.md